extensions/linz/README.md

```diff
@@ -62,22 +67,28 @@ See [ISO/IEC 13249-3:2016(en)](https://www.iso.org/obp/ui/#!iso:std:60343:en) fo
-| providers                      | Provider Object      | **REQUIRED**. The object provides information about a provider. A provider is any of the organizations that captures or processes the content of the assets and therefore influences the data offered by the STAC implementation. See [Provider Object](#provider-object).                                                                       |
-| linz:security_classification   | string               | **REQUIRED**. New Zealand Government [Security Classification](https://www.digital.govt.nz/standards-and-guidance/governance/managing-online-channels/security-and-privacy-for-websites/foundations/classify-information/). Must be one of `unclassified`, `in-confidence`, `sensitive`, `restricted`, `confidential`, `secret` or `top-secret`. |
-| processing:software            | Map<string, string>  | Recommended. The software and versions which were used to generate the dataset. See [reference](https://github.com/stac-extensions/processing).                                                                                                                                                                                                  |
-
-### LINZ Provider Object
+| Field Name                   | Type                 | Description                                                                                                                                                                                                                                                                                                                                      |
+| ---------------------------- | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| title                        | string               | **REQUIRED**. Collection title.                                                                                                                                                                                                                                                                                                                  |
+| linz:geospatial_type         | \[string]            | **REQUIRED**. A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types).                                                                                                                                                                                     |
```

Comment:
I can't think of a dataset / collection that we hold that would need to be categorised by multiple geospatial types, I guess because we tend to divide datasets / collections based on their "geospatial type". If a summary of all of the geospatial types of the items within the collection is required, then that would seem better implemented as item properties + summaries, but I see this more as a general categorisation which just has one geospatial type.

Reply:
That's a good point, and it sounds like the documentation cleanup is working if it highlights deficiencies :grin:. Do you want to create a separate issue for that though? It seems like it's out of scope of this PR.

Reply:
I raised the issue that `linz:geospatial_type` should be a Collection Field not an Item Field / Item Property for my understanding of its purpose and use. @billgeo suggested leaving it optionally as an Item Property which is fine but not the way I would be implementing it (we could again take the approach of waiting until we find a dataset where it doesn't work as a Collection Field - again - I can't think of an example).

Reply:
After discussion, to keep it simple, let's just leave it as only a collection field for now (remove from items). And change it to be a single string, rather than an array.

Reply:
If we ever need this for items/assets, we can get into that later when we need it.

Reply:
Had a chat with @billgeo and we went with this plan:

- [ ] Remove `linz:geospatial_type` from item fields, because we don't really need it yet and we don't know what the semantics of having values here which are different from the catalogue value would be.
- [ ] Make `linz:geospatial_type` a mandatory `string` (rather than non-empty array of `string`) type in the collection.